src/Limiters/limiter.jl

```diff
@@ -29,99 +29,110 @@ function quasimonotone_limiter!(
     max_ρq;
     rtol,
 )
-    space = axes(ρ)
+    if typeof(size(min_ρq)) == Tuple{Int}
```

Comment:
```suggestion
    if ndims(min_ρq) == 1
```
I'm wondering if this is simpler

Reply:
Yes, thank you!